packages/common/test/directives/ng_optimized_image_spec.ts

```
@@ -264,6 +264,47 @@ describe('Image directive', () => {
     });
   });
 
+  describe('loading attribute', () => {
+    it('should override the default loading behavior for non-priority images', () => {
+      setupTestingModule();
+
+      const template = '<img rawSrc="path/img.png" width="150" height="50" loading="eager">';
+      const fixture = createTestComponent(template);
+      fixture.detectChanges();
+
+      const nativeElement = fixture.nativeElement as HTMLElement;
+      const img = nativeElement.querySelector('img')!;
+      expect(img.getAttribute('loading')).toBe('eager');
+    });
+
+    it('should override the default loading behavior for priority images', () => {
+      setupTestingModule();
+
+      const template =
+          '<img rawSrc="path/img.png" width="150" height="50" loading="lazy" priority>';
```

It seems to me that we should throw an error in this case. If generally we're suggesting LCP images are "priority", it doesn't make sense for them to be lazy loaded. Why preload/preconnect them and set high priority if you're also lazy loading?